app/models/concerns/orchestration/tftp.rb

```diff
@@ -145,19 +149,19 @@ def queue_tftp_update
     if mac != old.mac
       set_tftp = true
       # clean up old TFTP reservation file
-      if old.tftp?
-        queue.create(:name => _("Remove old TFTP Settings for %s") % old, :priority => 19,
-                     :action => [old, :delTFTP])
-      end
+      queue_tftp_destroy(false, 19) if old.tftp?
```

Comment:
This replaces `old` with `self` in the value of the `:name`-key. I think it reads old to show the old value when the hostname is changed. I think, we should just live with the duplicate code.